homeassistant/components/pvoutput/config_flow.py

```diff
@@ -50,6 +50,7 @@ async def async_step_user(
             except PVOutputAuthenticationError:
                 errors["base"] = "invalid_auth"
             except PVOutputError:
+                LOGGER.exception("Cannot connect to PVOutput")
```

Comment:
Normally we only include the stack race for unknown exceptions.

Reply:
Sure, but how would one figure out "why" it cannot connect in this case?